scripts/checkpoint_converters/convert_gpt_nemo_to_mcore.py

```diff
@@ -40,22 +40,22 @@
 
 Then, run this conversion script:
 python convert_gpt_nemo_to_mcore.py \
- --input_name_or_path <path to extracted, TP1 PP1 legacy checkpoint folder> \
- --output_path <path to output nemo file>
+ --input-name-or-path <path to extracted, TP1 PP1 legacy checkpoint folder> \
+ --output-path <path to output nemo file>
 """
 
 
 def get_args():
     parser = ArgumentParser()
     parser.add_argument(
-        "--input_name_or_path",
+        "--input-name-or-path",
```

Comment:
can you don't change this to dash, keep it as underscore

Reply:
I changed the underscores to dashes to be consistent with the two other options `--cpu-only` and `--ignore-if-missing`. Besides, in command-line interfaces, the convention is to use hyphens (-) rather than underscores (_) to separate words in option names. It aligns with the usage in Unix-style commands, where hyphens are used as option prefixes.

Reply:
hi we should probably change others then. We try to keep the core args the same (using _) across the checkpoint_converters folder.